pandas_gbq/gbq.py

```diff
@@ -300,14 +315,41 @@ def load_user_account_credentials(self):
 
         return _try_credentials(self.project_id, credentials)
 
+    def get_default_credentials_path(self):
+        """
+        Gets the default path to the BigQuery credentials
+
+        .. versionadded 0.3.0
+
+        Returns
+        -------
+        Path to the BigQuery credentials
+        """
+
+        import os
+
+        if os.name == 'nt':
+            config_path = os.environ['APPDATA']
+        else:
+            config_path = os.path.join(os.path.expanduser('~'), '.config')
+
+        config_path = os.path.join(config_path, 'pandas_gbq')
```

Comment:
+1 to not clobbering `gcloud` credentials. The token we save doesn't have the full Google Cloud Platform scope so that would break other applications that depend on those credentials.

If folks set up application default credentials it won't hit this code anyway.